intro/notebooks-intro/notebooks-intro.ipynb

```diff
@@ -0,0 +1,311 @@
+    "Alternatively, you can double click the cell.\n",
+    "\n",
+    "To render the cell, press `Ctrl+Enter`.\n",
+    "You can also render the cell and advance to the next cell by pressing `Shift+Enter`.\n",
+    "If no next cell exists, then pressing `Shift+Enter` adds a new cell at the bottom of the notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CHANGE_ME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "2. Edit the following code cell to print the message *Hello Red Hat*.\n",
+    "Press `Ctrl+Enter` to render the result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Hello!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Execute the following cell several times to verify that it returns the same *Process Identifier* (PID) number.\n",
+    "This is the PID of the Python kernel. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.getpid()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4. Use a cell to assign the `hey` string to a variable called `test`, and print the value in a different cell.\n",
+    "Verify that the cell which prints the value renders the string\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "test = \"hey\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Display the `redhat.png` image in the notebook's directory by using markdown.\n",
+    "To do this, replace the `CHANGE_ME` string in the image name.\n",
+    "\n",
+    "![Red Hat logo](CHANGE_ME.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "6. Display an image by using Python code.\n",
+    "For this, replace the `CHANGE_ME` string with the `redhat.png` file name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from IPython import display\n",
+    "\n",
+    "display.Image(\"CHANGE_ME\", width=600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "7. Uncomment the last two lines from the following source code and run the cell to plot a sine graph.\n"
```

Comment:
There are no commented lines in the following cell.

Reply:
Sorry, that was me doing the exercise

Reply:
[a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)